### ISSUE TITLE
5.4 download links in readme lead to 404

### ISSUE THREAD

**Issue by idrougge** [uraimo/buildSwiftOnARM#69]
The readme file has links to download prepackaged Swift compilers for various distributions, but the links seem to be broken.

**Comment by uraimo** [comment]
Yep, I still have to build the full set with my docker setup, it takes approximately 60 hours now, I will do a proper 5.4 [release](https://github.com/uraimo/buildSwiftOnARM/releases) when I get around at doing it.
Right now you could build the single release you need, no more than 7h.

**Comment by idrougge** [comment]
Sorry about that, keep up the good work!

**Comment by dilames** [comment]
Any updates about 5.4 build?

**Comment by tomieq** [comment]
I keep my fingers crossed we will get Swift 5.4 for download soon. Thanks in advance!
